docs/scripts/helper-scripts.rst

```diff
@@ -13,8 +13,8 @@ Build
 
    make-all
 
-ES2K Build
-----------
+ACC Build
```

Comment:
Suggestion: Since the helper-scripts.rst is part of general documentation, it will be good to have something that indicates that ACC build is applicable only for ES2K target.

Maybe change it to ES2K ACC build?

Reply:
I'll do you one better. Instead of changing the heading, I'll add an explanatory sentence that says what ACC means. :)